scripts/entities/entity.gd

```diff
@@ -0,0 +1,3 @@
+extends Node2D
+class_name Entity
+var map_pos: Vector2i
```

Comment:
Smart, might be a couple of other global things we can throw in here (stuff like id, name, etc) not sure tho. We'll deliberate on this at some point :3

Reply:
I was going to throw health, and some basic sub-systems in there (dynamic lighting, etc.) but decided against it as this was simply a turn base pr. we also have a plan to use it for a save system.